activitypub.php

```diff
@@ -19,6 +19,7 @@
  * Initialize plugin
  */
 function init() {
+	\defined( 'ACTIVITYPUB_NUMBER_IMAGES' ) || \define( 'ACTIVITYPUB_NUMBER_IMAGES', 3 );
```

Comment:
what about `ACTIVITYPUB_NUMBER_POST_IMAGES` or something similar (I am no native speaker) to be more specific about the usecase?

Reply:
Yeah, that name is not clear.  I changed it to `ACTIVITYPUB_MAX_IMAGE_ATTACHMENTS`.  I'm not sure whether attachments can also show up in comments, or if they can only be in posts?

Reply:
shouldn't this simply work? https://github.com/pfefferle/wordpress-activitypub/pull/248/commits/6992fbbe226eb1753d34637cb50c10b9009d778c